src/user-icon/classes.scss

```diff
@@ -2,18 +2,19 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
 
-.iui-user-icons-small {
-  @include iui-user-icons-small;
-}
+.iui-user-icon {
+  @include iui-user-icon;
+  @include iui-user-icon-size;
 
-.iui-user-icons-medium {
-  @include iui-user-icons-medium;
-}
+  &.iui-small {
+    @include iui-user-icon-size($size: $iui-icons-large, $font-size: $iui-font-size-small);
+  }
 
-.iui-user-icons-large {
-  @include iui-user-icons-large;
-}
+  &.iui-large {
+    @include iui-user-icon-size($size: $iui-icons-xxl, $font-size: $iui-font-size-subheading);
+  }
 
-.iui-user-icons-x-large {
-  @include iui-user-icons-x-large;
+  &.iui-xl {
```

Comment:
Not sure I like this one. Could be confused with the `$iui-xl` variable.

Reply:
Changed from `xl` to `x-large`.